index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <title>Fork Me! FCC: Test Suite Template</title>

```

Comment:
Please change your title to the template name you are working on.
<title>Product Landing Page</title>

Reply:
Good job on the head section. Please change your title to the template name that you are working on, so that when the page loads the title loads your template name. Hence, everyone can understand the project you are working on.
`<title>Product Landing Page</title>`